tools/reviewer/web/src/app/dashboard/reviews-panel/reviews-panel.component.ts

```diff
@@ -39,23 +35,34 @@ export class ReviewsPanelComponent implements OnInit {
         // and create Diff from proto and categorize
         // into a specific list.
         for (const diff of diffs) {
-          for (const reviewer of diff.getReviewerList()) {
-            if (reviewer.getEmail() === this.authService.userEmail) {
-              if (reviewer.getNeedsattention()) {
-                // Need attention of user
-                this.diffGroups[Lists.NeedAttention].push(diff.toObject());
-              } else {
-                // User is cc'ed on this
-                this.diffGroups[Lists.CcedOn].push(diff.toObject());
-              }
-            } else if (reviewer.getEmail() === this.authService.userEmail) {
-              if (diff.getStatus() === Diff.Status.REVIEW_NOT_STARTED) {
+          diff.getDescription();
+          ///
+          const you: string = this.authService.userEmail;
```

Comment:
you is you or me? or me is you?
How about we call it `user` or `currentUser`?
I have a friend who wrote a server that printed "My bad" and "Your bad" on errors when it was the server's fault or the caller's fault. It was funny but kinda confusing.

Reply:
xD